src/app/modules/submission/encrypt-submission/encrypt-submission.middleware.ts

```diff
@@ -319,7 +333,7 @@ export const encryptSubmission = async (
 
   req.formsg.encryptedPayload = {
     attachments: encryptedAttachments,
-    responses: filteredResponses.value as EncryptFormFieldResponse[],
+    responses: req.formsg.filteredResponses,
```

Comment:
🤔 was the intent here originally to check if the array is empty? (empty array is truthy so this check is indeed redundant)